chapters/10-community-involvement.Rmd

```diff
@@ -1,37 +1,64 @@
 # Involving the community {#community-involvement}
 
 - Designing your package to be used by or contributed to from the community
-  (refer to Mozilla Science Labs Open Leaders material)
 - Being inclusive (code of conduct)
 - Contributing guidelines
 - Licensing
-- Governance
+- Project management for a user-friendly package
 - NEWS file
 - reprex
 
+## Learning objectives
+
+* Edit a project README and NEWS file to be welcoming to contributors and users
+* Develop a code of conduct and create CODE_OF_CONDUCT.md
```

Comment:
These could have a bit more "pedagogy" related to them. E.g. "To understand the role of having explicit Codes of Conduct and to create one relevant to a project".